hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/condition/EventCondition.java

```diff
@@ -31,48 +31,63 @@
 public class EventCondition extends Condition {
```

Comment:
I would like to see more class-level and maybe field level jdoc on how this fairly complex condition is used.  The processExpression jdoc is good but that is a private method.